tests/testthat/test_variant_set.R

```diff
@@ -44,25 +44,44 @@ test_that("SMMAT matches burden", {
 	nullmod <- .testNullmod(n, MM=TRUE)
         smmat <- .testVariantSetSMMAT(nullmod, geno, weights)
         burden <- .testVariantSetBurden(nullmod, geno, weights, burden.test="Score")
+        expect_equal(smmat$Score_burden,burden$Score)
+        expect_equal(smmat$Score.SE_burden, burden$Score.SE)
+        # Burden test does not return a test statistic.
```

Comment:
@amstilp -- I think it does. It's `Score.Stat`. But it's the sqrt of what you had implemented in the SMMAT version

Reply:
Wow, how did I miss that 🤦‍♀️